src/__test__/example.test.ts

```diff
@@ -1,10 +1,14 @@
-console.clear()
+describe('example', () => {
+   it('should be true', () => {
+      expect(true).toBe(true)
+   })
+})
 
 /***************************************
  * Preparando el entorno con Jest
  ***************************************/
```

Comment:
I'd just changed all your comments from Spanish to English.

Reply:
This file is of a jest's course. :sweat_smile: